src/editmap.cpp

```diff
@@ -1836,8 +1835,8 @@ int editmap::mapgen_preview( real_coords &tc, uimenu &gmenu )
 
                 } else if( gpmenu.ret == 3 ) {
                     popup( _( "Changed oter_id from '%s' (%s) to '%s' (%s)" ),
-                           orig_oters.t().name.c_str(), orig_oters.c_str(),
-                           omt_ref.t().name.c_str(), omt_ref.c_str() );
+                           orig_oters.obj().name.c_str(), orig_oters.id().c_str(),
```

Comment:
Not so many (~80), but in various parts of the code, so I think it deserves a separate PR.